components/javascript/toast.js

```diff
@@ -0,0 +1,31 @@
+var topToast = document.querySelector("#top-toast-id");
+var topToastContent = document.querySelector('#top-content');
+var topToastClose = document.querySelector('#top-toast-close');
+
+var bottomToast = document.querySelector('#bottom-toast-id');
+var bottomToastContent = document.querySelector('#bottom-content');
+var bottomToastClose = document.querySelector('#bottom-toast-close');
+
+function openTopToast(){
```

Comment:
I suggest you to write all functions as arrow functions.

Reply:
Updated code in ES6.Thank you for the review